setup/vagrant/provision.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+
+cd /opt/redash/current
+cp /opt/redash/.env /opt/redash/current
+cd /opt/redash/current/rd_ui
+bower install
+cd /opt/redash/current
+
+#install requirements
+sudo pip install -r /opt/redash/current/requirements_dev.txt
+sudo pip install -r /opt/redash/current/requirements.txt
+sudo pip install pymongo==3.2.1
```

Comment:
Running requirements_all_ds.txt has some dependencies that are not able to be installed. I could spend some time trying to fix all the dependencies. But they are not really required for anything on the vagrant box as far as I can tell. So I just decided to go with installing pymongo, which is the only thing we really need (one test fails when we dont have it). I could be wrong though? The dependencies can be sorted out in a later PR? Is not using requirements_all_ds.txt introducing a regression?